src/rcn_py/orcid.py

```diff
@@ -0,0 +1 @@
+"""Functionality to search through orcid database"""
```

Comment:
ORCID should be uppercase

```suggestion
"""Functionality to search through ORCID database"""
```